lib/helper/firefoxProfileCreator.py

```diff
@@ -72,7 +72,7 @@ def _install_profile_extensions(self, extensions_settings={}):
         return
 
     def remove_firefox_profile(self):
-        if os.path.isdir(self._firefox_profile_path):
+        if self._firefox_profile_path and os.path.isdir(self._firefox_profile_path):
```

Comment:
I think self._firefox_profile_path was initialed as an empty string. Therefore, os.path.isdir('') will return False and checking of self._firefox_profile_path would be unnecessary?